scripts/build/arch/alpha.sh

```diff
@@ -4,3 +4,16 @@ CT_DoArchTupleValues () {
     # The architecture part of the tuple:
```

Comment:
At the time @mingwandroid committed this, eglibc was still in the tree, I think. His branch has been sitting for quite some time. I'll change the commit message, but I think it's nitpicking :)